spec/interactors/destroy_map_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "rails_helper"
+
+describe DestroyMap do
+  it "destroys the map" do
+    campaign = instance_double(Campaign, current_map: nil)
+    map = instance_double(Map, destroy: true)
+    change_current_map = instance_double(ChangeCurrentMap, call: nil)
```

Comment:
Would these doubles actually be called in the interactor? They're not spying on the class itself

Reply:
Good catch, fixed in 10042df